src/eredis_cluster_client.erl

```diff
@@ -383,6 +383,12 @@ qa(State, Command) ->
     Transaction = fun(Worker) -> qw(Worker, Command) end,
     [eredis_cluster_pool:transaction(Pool, Transaction) || Pool <- Pools].
 
+
+-spec qs(State::#state{}, redis_command()) -> redis_result().
+qs(Pool, Command) ->
+    Transaction = fun(Worker) -> qw(Worker, Command) end,
+    eredis_cluster_pool:transaction(Pool, Transaction).
```

Comment:
- spec says state.. argument says pool ?
- why do we need this function? looks like we are basically passing the exact worker name and the command.
If we just use q(..) then it will anyways determine the worker and allot the work automatically.. Am I missing something?
- qa looks useful to me and I'm guessing that is already exposed??

Reply:
Will fix the spec,
I need this function to execute the scan command from the other diff on particular RedisMaster (Pool). If you use q() you redis command needs a key to hash. Scan does not really have a key.

For qa it is not exposed until you add it to the gen_server handle_call.